### ISSUE TITLE
ScatterplotLayer circles not rounded correctly

### ISSUE THREAD

**Issue by ilijapuaca** [UnfoldedInc/deck.gl-native#126]
From #124:
> That being said I don't think these look the way they're supposed to look, they seem to be squares instead of circles that I'm seeing on the web. Also, when zooming in very closely, they seem to stop scaling at some point. I'm seeing weird geometry in the geometry inspector (could just be a bug in the tool, it's pretty flimsy sometimes) although all of the values seem valid, no NaNs or anything.

> Changing SMOOTH_EDGE_RADIUS in geometry.glsl.h to 0 seems to completely get rid of the issue. I am not clear on the specifics about smoothedge/smoothstep, I see that it's a way to interpolate values, but I am not sure why it'd cause results we're seeing when it has the same value in JS.

**Comment by ibgreen** [comment]
I believe the smoothstep is there to minimize pixelization/jaggedness of circle. Maybe the resulting alpha channel interacts with blending?

**Comment by ilijapuaca** [comment]
Blending wasn't in place when I initially worked on this. Just gave it a try and the circles look perfect.

I was thrown off by that last line in fragment shader as I thought `inCircle` would either be `0` or `1`, and since we already discard the ones with value `0` I wasn't sure what it was supposed to do, as I tried commenting it out and did not make a difference.

Closing this as the issue seems to be fixed
